doc/iterations/2024.5/shapes/guarantee_view_models_compatibility_between_versions.adoc

```diff
@@ -0,0 +1,39 @@
+= Add the support for view models migration
+
+== Context
+
+Users want to upgrade the version of Sirius Web without losing the models they created with an older version.
+
+== Key Result
+
+When a user creates a view model with a version n of Sirius Web, the model created must be available with any version>n of Sirius Web. 
```

Comment:
Not specific to view models, it should work with domain models, SysML, custom ones like Flow or anything else. View will juste be one of our examples.